.cron.yml

```diff
@@ -15,4 +15,5 @@ jobs:
           type: decision-task
           target-tasks-method: tab_performance_test
           treeherder-symbol: tab-performance-test
-      when: [{weekday: 'Sunday', hour: 22, minute: 0}]
\ No newline at end of file
+      when: 
+        - {hour: 22, minute: 0}
```

Comment:
@AaronMT I have looked at the cron.yml for android and looks like this is the format for daily job, could you please confirm if you know? Thanks!

Reply:
For reference https://firefox-source-docs.mozilla.org/taskcluster/cron.html